strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/AuthenticatorsRegistrySerializer.java

```diff
@@ -0,0 +1,43 @@
+package org.carlspring.strongbox.authentication.registry.support;
+
+import org.carlspring.strongbox.authentication.api.Authenticator;
+import org.carlspring.strongbox.authentication.registry.AuthenticatorsRegistry;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public class AuthenticatorsRegistrySerializer
```

Comment:
As you may have seen, across the project we're using JAXB for serializing/deserializing XML. This is normally handled using the `GenericParser` class. I think it would be best to replace this code with it, (unless I'm missing something).